modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -6,192 +6,236 @@
 [abstract]
 The Couchbase Python SDK allows Python applications to access a Couchbase cluster.
 The Python SDK offers a traditional synchronous API as well as integration with _twisted_, _gevent_, and _asyncio_.
-It depends on the C SDK, _libcouchbase_, which it uses for performance and reliability.
+It depends on the C SDK, _libcouchbase_, (included automatically) which it uses for performance and reliability.
 
 // tag::prep[]
 
 The Couchbase Python SDK 3.0 is a complete rewrite of the API, reducing the number of overloads to present a simplified surface area, and adding support for future Couchbase Server features like Collections and Scopes (available in Couchbase Server 6.5 as a xref:concept-docs:collections.adoc[developer preview]).
-The 3.0 Python SDK introduces comprehensive https://www.python.org/dev/peps/pep-0484/[PEP-484] style type annotations.
 
+The 3.0 Python SDK introduces comprehensive https://www.python.org/dev/peps/pep-0484/[PEP-484] style type annotations.
 
 
 == Requirements
 
+The Python SDK 3.0 requires Python 3, with Python 3.5 and above supported.
+
 Couchbase Python SDK bundles libcouchbase automatically, so no need to install it separately.
-You may need cmake to install, although the installer will attempt to download it from PyPI automatically.
 
-The Python SDK 3.0 requires Python 3, with Python 3.5 and above supported.
+You may need cmake to install, although the installer will attempt to download it from PyPI automatically.
 
 // end::prep[]
 
 // tag::install[]
 
-[NOTE]
-====
-Currently the Python Client source distribution requires the OpenSSL headers and libraries that the Python client itself was built against to be installed prior to the client itself for TLS support to be provided. 
-Additionally the installer relies on PEP517 which older versions of PIP do not support. 
-If you experience issues installing it is advised to upgrade your PIP/setuptools installation as follows: 
-[source,console]
-----
-$ python3 -m pip install --upgrade pip setuptools wheel
-----
-====
-
+NOTE: Currently the Python Client requires the OpenSSL headers and libraries that the Python client itself was built against to be installed prior to the client itself. 
```

Comment:
Revert, this must be based on an older document.

Reply:
please make this change as you see fit. I have no opinion on this part.